sections/collapsible-content.liquid

```diff
@@ -0,0 +1,382 @@
+{{ 'component-accordion.css' | asset_url | stylesheet_tag }}
+{{ 'collapsible-content.css' | asset_url | stylesheet_tag }}
+
+<div class="collapsible-content layout-{{ section.settings.layout | split: '_' | first }}-container color-{{ section.settings.color_scheme }}{% if section.settings.layout == 'section_container' %} page-width{% endif %}">
+  <div class="collapsible-content-wrapper{% if section.settings.layout == 'section_container' %} color-{{ section.settings.container_color_scheme }}{% endif %}">
+    <div class="{% if section.settings.image == blank %} collapsible-content-wrapper--narrow{% else %} page-width{% endif%}">
+      <div style="text-align: {{ section.settings.heading_alignment }}">
+        {%- if section.settings.caption != blank -%}
+          <p class="caption-with-letter-spacing"> {{ section.settings.caption }}</p>
+        {% endif %}
+        {%- if section.settings.heading != blank -%}
+          <h2 class="h1">{{ section.settings.heading }}</h2>
```

Comment:
I would also like to get designer's feedback on the size of this heading. Imo it's too big compared to other sections' heading. 

I can see how it's as big as for example the image with text but I think the layout is different there and it makes sense.
In this section layout it looks like more like the `feat. collection` for example and should match the heading size 🤔 

![](https://screenshot.click/21-12-uehpq-53415.png)

Reply:
I believe @wiktoriaswiecicka had been working on adding a `Heading size` dropdown to every section. I think it got descoped so we can reduce it to h2 for the time being. 👍

Reply:
Sweet, now using a h2 style to match other sections heading.